src/d3a/constants.py

```diff
@@ -38,7 +38,13 @@
 
 SIMULATION_PAUSE_TIMEOUT = 600
 
-D3A_TEST_RUN = False
+# Controls whether the past markets are still being kept in the simulation memory for the duration
+# of the simulation. Helpful in the unit / integration tests, since some of these rely on the
+# markets remaining in-memory until the end of the simulation run.
+# Is also needed for the raw JSON files and the plots when running in CLI mode.
+# Also helpful when debugging, in order for the interpreter to have access to all markets that a
+# simulation has ran through.
```

Comment:
For completenes, can you please add, that the bids and offers of all markets are buffered in the BaseStrategy when set to True, too? Or do you think this is clear?

Reply:
Makes sense, I'll add it too. Thanks!